qa/integration/services/service_container.rb

```diff
@@ -60,4 +60,24 @@ def stop_container
     @container.stop
     @container.delete(:force => true, :volumes => true)
   end
+
+  # Get environment variable from container's ENV.
+  def envvar(var)
+    # Exec command returns an array with [[stdout], [stderr], exit_code] layout.
+    res = @container.exec(['printenv', var])
+    res[0].join(" ").chomp
+  end
+
+  # Copy certificates from container to directory in the host.
+  def certificates(dest_dir)
```

Comment:
@wainersm we're doing this, because we want to use the cert files with the Logstash that is running on the host right? (sorry this is probably an obvious question :))